lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
Yeah sorry, I was asking whether you needed another flag.  Is verbose not enough?